server/server.go

```diff
@@ -360,6 +361,7 @@ func (s *Server) ListenAndServe() error {
 // UnixListenAndServe listens on the Unix socket address and handles keyless requests.
 func (s *Server) UnixListenAndServe() error {
```

Comment:
I found the test leaks the unix socket. But I haven't figured out a way to contain that.